test/test_cli.py

```diff
@@ -1,8 +1,7 @@
 import unittest
 from unittest.mock import patch
 
-from .util.cli_helpers import cli_arg_test, cli_test
-
+from .util.cli_helpers import cli_test, cli_arg_test
```

Comment:
. at start of import

Reply:
are you suggesting `from .util.cli_helpers import .` instead?